src/dispatchers/mouseDispatcher.ts

```diff
@@ -92,6 +115,11 @@ export module Dispatcher {
       mrBCR = this._measureRect.getBoundingClientRect();
       var testPoint = { x: mrBCR.left, y: mrBCR.top };
 
+      // invalid measurements -- SVG might not be in the DOM
+      if (origin.x === testPoint.x || origin.y === testPoint.y) {
+        return { x: null, y: null };
```

Comment:
Yeah this is definitely safe, but I think it's cleaner to return null, so that the check above is simply
if (newMousePosition == null)

Would there be a situation where x could be null and y isn't and this would be an invalid mousePosition?  We could just simply just report all of this as a null point